cmd/gitbase/command/server.go

```diff
@@ -49,14 +51,11 @@ type Server struct {
 	DisableSquash bool     `long:"no-squash" description:"Disables the table squashing."`
 	TraceEnabled  bool     `long:"trace" env:"GITBASE_TRACE" description:"Enables jaeger tracing"`
 	ReadOnly      bool     `short:"r" long:"readonly" description:"Only allow read queries. This disables creating and deleting indexes as well." env:"GITBASE_READONLY"`
-	// SkipGitErrors disables failing when Git errors are found.
-	SkipGitErrors bool
-	// Version of the application.
-	Version string
 
-	engine *sqle.Engine
-	pool   *gitbase.RepositoryPool
-	name   string
+	SkipGitErrors bool // SkipGitErrors disables failing when Git errors are found.
```

Comment:
This is changing the output of `gitbase server --help` command? If yes, could you change it on the documentation too please?

Reply:
@ajnavarro - done